cli/enry/main.go

```diff
@@ -140,3 +145,28 @@ func writeStringLn(s string, buff *bytes.Buffer) {
 	buff.WriteString(s)
 	buff.WriteByte('\n')
 }
+
+func splitArgs() (args []string, flagArgs []string) {
+	for _, arg := range os.Args {
+		if strings.HasPrefix(arg, "-") {
+			flagArgs = append(flagArgs, arg)
+		} else {
+			args = append(args, arg)
+		}
+	}
+	return
+}
+
+func parseFlags(flags []string) (jsonFlag bool, breakdownFlag bool, helpFlag bool) {
+	for _, flag := range flags {
+		switch {
+		case "-help" == flag || "-h" == flag:
```

Comment:
I think linguist doesn't have a help flag, it prints the usage always it finds something unexpected (an unknown flag or a wrong path)